src/FunctionApps/python/tests/IntakePipeline/conversion_test.py

```diff
@@ -233,10 +185,13 @@ def test_log_generic_error(mock_log, mock_fhir_post):
     )
 
     mock_log.assert_called_with(
-        "Error during $convert-data -- HTTP Code: 400, Response Content some-error"
+        "HTTP 400 code encountered on $convert-data for some-filename-0"
     )
 
-    assert response == {}
+    assert response == {
```

Comment:
I don't think we need to assert that the call from `convert_message_to_fhir` is correct here as that's not the point of this test.